types/src/proof/definition.rs

```diff
@@ -602,10 +602,57 @@ impl SparseMerkleRangeProof {
         Self { right_siblings }
     }
 
-    /// Returns the siblings.
+    /// Returns the right siblings.
     pub fn right_siblings(&self) -> &[HashValue] {
         &self.right_siblings
     }
+
+    /// Verifies that the rightmost known leaf exists in the tree and that the resulting
+    /// root hash matches the expected root hash.
+    pub fn verify(
+        &self,
+        expected_root_hash: HashValue,
+        left_siblings: Vec<HashValue>,
```

Comment:
it's the left siblings of "the rightmost known leaf", instead of "the range", which is a bit confusing. Probably document it, and/or switch position with `rightmost_known_leaf`.

Reply:
Yeah, agreed. I'll add a comment to the top of the struct and the verify method to make this distinction 😄